examples/data/e2e_tests/test_jsons/castingConformanceRuleTest.json

```diff
@@ -2,9 +2,9 @@
 {
     "vars": {
       "spark-submit": "spark-submit --num-executors 2 --executor-memory 2G --deploy-mode client",
-      "spark-conf": "--conf 'spark.driver.extraJavaOptions=-Denceladus.rest.uri=http://localhost:8080/rest_api/api -Dspline.mongodb.name=spline -Dspline.mongodb.url=mongodb://127.0.0.1:27017/ -Denceladus.recordId.generation.strategy=stableHashId'",
-      "enceladus-job-jar": "spark-jobs/target/spark-jobs-2.19.0-SNAPSHOT.jar",
-      "credentials": "--rest-api-credentials-file ~/.ssh/menasCredential.properties",
+      "spark-conf": "--conf 'spark.driver.extraJavaOptions=-Denceladus.rest.uri=http://localhost:8080/rest_api_war/api -Dspline.mongodb.name=spline -Dspline.mongodb.url=mongodb://127.0.0.1:27017/ -Denceladus.recordId.generation.strategy=stableHashId'",
+      "enceladus-job-jar": "spark-jobs/target/spark-jobs-3.0.0-SNAPSHOT.jar",
+      "credentials": "--rest-api-credentials-file ~/.ssh/menas-credential.properties",
```

Comment:
Why this change? Seems like personal setup.

Reply:
I would like to see it as small improvement to reach same pattern where CamelCase is used for Class files only. I am used to write these non-class files in format "word-word" or "word_word". I can see more similar locations with this naming. Are we following some format rules when to use CamelCase, w-w and w_w?